Java/Java.sublime-syntax

```diff
@@ -586,7 +586,7 @@ contexts:
     - include: invalid-generic-tags
   throws:
     - match: \bthrows\b
-      scope: storage.modifier.java
+      scope: keyword.declaration.throws.java
```

Comment:
Pretty sure `keyword.control.flow.throws` makes more sense here.

`keyword.declaration` is only used for `extends` basically: https://github.com/sublimehq/Packages/search?utf8=%E2%9C%93&q=%22keyword.declaration%22&type=.

See also https://github.com/sublimehq/Packages/issues/1228.